src/current/_includes/v24.3/backups/old-syntax-removed.md

```diff
@@ -0,0 +1,5 @@
+{{site.data.alerts.callout_danger}}
+The `BACKUP ... TO` and `RESTORE ... FROM` syntax has been removed from CockroachDB v24.3 and later.
```

Comment:
Oh one small nit, but perhaps we should say `RESTORE ... FROM <backup_uri>` or something like that? Since both the old syntax and new syntax have `RESTORE ... FROM`, it might be a little confusing.

Reply:
Yes, very good point. Added!